src/components/Grid/_grid.scss

```diff
@@ -0,0 +1,56 @@
+// ----------------------------------
+// UI Components
+// Grid
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+/* Settings */
+
+@import 'grid-settings';
+
+/* Mixins */
+
+@import 'grid-mixins';
```

Comment:
Thinking it could be an idea to start creating a `styles` folder within each component? Would allow more stylistically complex components to be tidier.

Reply:
agreed, keep it as a single named scss file ordinarily and expand to styles folder if needed